lib/nulldb/core.rb

```diff
@@ -3,6 +3,9 @@
 require 'active_record/connection_adapters/nulldb_adapter'
 
 module NullDB
+  LEGACY_ACTIVERECORD = 
+    (ActiveRecord::VERSION::MAJOR < 5) && (ActiveRecord::VERSION::MINOR < 2)
```

Comment:
Would this mean ActiveRecord 4.1 is legacy, but 3.2 is not (because it's minor is not less than 2)